keyboards/tweetydabird/lbs6/info.json

```
@@ -0,0 +1,74 @@
+{
+	"keyboard_name": "Little Big Scroll 6",
+	"manufacturer": "Tweetys Wild Thinking",
+	"url": "https://lectronz.com/stores/tweetys-wild-thinking",
+	"maintainer": "TweetyDaBird",	
+	
+	"bootloader": "caterina",
+	"processor": "atmega32u4",
+	
+	"features": {
+		"bootmagic": true,
+		"command": false,
+		"console": false,
+		"extrakey": true,
+		"mousekey": true,
+		"nkro": false,
+		"rgblight": true,
+		"encoder": true
+	},
+
+  "build": {
+    "lto": true
+  },
+
+	"usb": {
+		"vid": "0xFEED",
+        "pid": "0x23B0",
+		"device_version": "1.1.0"
+	},
+	
+	"debounce": 5,
+	
+	"matrix_pins": {
+        "direct": [
+			[null, "F5", null],
+            ["D4", "D7", "B4"],
+			["C6", "E6", "B5"]
+        ]
+    },
+
+	"encoder": {
+        "rotary": [
+            { "pin_a": "F7", "pin_b": "F6", "resolution": 2 }
+        ]
+    },
+
+	"rgblight": {
+        "led_count": 8,
+        "pin": "D2",
+        "hue_steps": 8,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+		"max_brightness": 200,
+        "animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```